src/main/java/de/srendi/advancedperipherals/lib/peripherals/AutomataCorePeripheral.java

```diff
@@ -84,4 +83,25 @@ public void setAttribute(String attribute) {
     public Direction validateSide(String direction) throws LuaException {
         return super.validateSide(direction);
     }
+
+    public boolean isDestroyed() {
+        return this.destroyed;
+    }
+
+    public boolean canOverpowerAction() {
```

Comment:
I think `canActionOverpower` would fit a bit better, I didn't really get what you've meant by that before I checked the src
Or maybe a small comment/javadoc